src/Connector.SqlServer/Utils/StringExtensions.cs

```diff
@@ -9,7 +9,15 @@ internal static class StringExtensions
         /// </summary>
         public static string ToSanitizedSqlName(this string value)
         {
-            return Regex.Replace(value, @"[^_A-Za-z0-9]+", string.Empty);
+            var withoutSpecialCharacters = Regex.Replace(value, @"[^_A-Za-z0-9]+", string.Empty);
+
+            if (withoutSpecialCharacters.Length > 0 &&
```

Comment:
It's a good catch! Would empty string then considered a valid name? 🤔

Reply:
Technically yes, but it can't be inputted from frontend. You can't input anything but alphanumeric characters either, so technically the check is redundant.

I would really like to be able to return a more rich result in `GetValidContainerName` where this is used, but that would require a change to the interface, and UI.

Reply:
Shall we then throw an exception?

Reply:
Done